.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
```

Comment:
(Maybe) try adding [django-upgrade](https://github.com/adamchainz/django-upgrade) to the pre-commit config?

Reply:
Good idea, I added the pre-commit hook.

Reply:
Heads up @tobiasmcnulty and @Afani97, if the package you are upgrading needs to support more than one Django version `django-upgrade` may not be a good option since it overrides imports and method names to the target Django version. It happened a few times until I realized that the tool was doing it.